resources/i18n/pl_PL/uranium.po

```diff
@@ -356,7 +356,7 @@ msgstr "Nie znaleziono ważnych rozszerzeń OpenGL. Ten program wymaga wsparcia
 #, python-brace-format
 msgctxt "@info:status Don't translate the XML tag <filename>!"
 msgid "Cannot open files of the type of <filename>{0}</filename>"
-msgstr ""
+msgstr "No można otworzyć plików typu <filename>{0}</filename>"
```

Comment:
Isn't it "Nie można.."?

Reply:
Oh. Sorry. That's my mistake. Thanks for being patient.

Reply:
No problem. Looks like my Polish is still not too bad 😋